src/Movie/Movie.js

```diff
@@ -3,8 +3,8 @@ import { getMovie } from '../api-calls/apiCalls';
 import './Movie.css'
 
 class Movie extends Component {
-    constructor() {
-        super();
+    constructor(props) {
+        super(props);
```

Comment:
Props are directly passed in the constructor here, though I am unsure if this is necessary. I do not think it is hurting at least.